Releases/Manifests/5.3.0.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Can we add a comment here about what it is, as well as (in a comment) what the version of Firebase is?

Reply:
Since json doesn't allow comments, I added a README